7_drivers_munge/src/GCM_driver_nc_utils.R

```diff
@@ -59,8 +59,19 @@ generate_gcm_nc <- function(nc_file, gcm_raw_files, vars_info, grid_params, spat
 
   # Set up attributes for NetCDF that are independent of variables
   data_time_units <- "days since 1970-01-01 00:00:00"
-  data_attributes <- list('title' = global_att)
-  data_attributes <- append(data_attributes, grid_params) # add grid parameters to list of global attributes
+  source_info <- list('title' = sprintf("Notaro debiased downscaled GCM data - %s", gcm_name),
+                      'source' = paste("These data were derived from a source raster dataset of dynamically downscaled",
+                                       "global climate model (GCM) data generated by Michael Notaro (mnotaro@wisc.edu)",
+                                       "and others (see https://ccr.nelson.wisc.edu/dynamical-downscaling/index.php).",
+                                       "For internal USGS users, the source dataset can be accessed at this url:",
```

Comment:
I think we should leave out the internal data url, since it isn't likely to remain hosted there into the future.